package.json

```
@@ -8,12 +8,20 @@
   "license": "Apache",
   "scripts": {
     "api": "node src/api/index.js",
-    "seed": "node src/seedFake.js"
+    "seed": "node src/seedFake.js",
+    "scan:current": "node src/scanner/scanCurrent.js",
+    "scan:old": "node src/scanner/nftOwnersAtBlock.js",
+    "seed:db": "node src/scanner/nftOwnersAtBlock.js && node src/scanner/scanCurrent.js"
```

not sure `scan:current` is useful
```suggestion
    "scan:fastforward": "node src/scanner/nftOwnersAtBlock.js",
    "scan:start": "node src/scanner/nftOwnersAtBlock.js && node src/scanner/scanCurrent.js"
```

---

have removed scan:current earlier